src/components/Charts/DonutChart/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { CSSProperties } from 'react';
+import { CHART_COLORS, theme } from '@/theme';
+
+export const chartStyles = {
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+  } as CSSProperties,

```

Comment:
You can create an interface for the styles object, this will prevent the need to use `as CSSProperties` every time